tasks/oracle.yml

```diff
@@ -15,3 +15,11 @@
 
 - name: Set Oracle Java as the default
   alternatives: name=java path="/usr/lib/jvm/java-{{ java_major_version }}-oracle/jre/bin/java"
+
+- name: Write ld.so.conf for java
+  template:
+    src: java.conf.j2
+    dest: /etc/ld.so.conf.d/java.conf
```

Comment:
I think we might be able to attach a:

```yaml
  notify:
    - Execute ldconfig
```

To this task, and then create an associated handler in `handlers/main.yml` with:

```yaml
---
- name: Execute ldconfig
  command: "ldconfig"
```

That should only fire when the contents of `java.conf` changes.